schema_utils_test.py

```diff
@@ -582,24 +662,61 @@ def test_notification_email_list_validator(self):
                         [u'admin@oppia.commmm'], [u'a@.com']]
         self.check_normalization(schema, mappings, invalid_vals)
 
-    def test_sanitize_url(self):
-        """Tests that sanitize_url method correctly sanitizes a string
-        representing a URL and raises error for invalid strings.
+    def test_normalize_spaces(self):
+        """Test static method normalize_spaces; should collapse multiple
+        spaces.
+        """
+        normalize_spaces = schema_utils.Normalizers.get('normalize_spaces')
+        self.assertEqual('dog cat', normalize_spaces('dog     cat'))
+        self.assertEqual('dog cat', normalize_spaces('  dog cat'))
+        self.assertEqual('dog cat', normalize_spaces(' dog   cat   '))
```

Comment:
May be add false test like: 'dog cat' != '  dogcat  ' ?

Reply:
Addressed in most recent commit!